test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -1001,6 +1001,14 @@ def test_group_norm(self):
         model = nn.GroupNorm(3, 6)
         self.run_model_test(model, train=True, input=c, batch_size=BATCH_SIZE)
 
+    def test_rsub(self):
+        class RsubMod(torch.nn.Module):
```

Comment:
RsubModel?

Reply:
Just noticed other "Mod" does not mean "Model" but means "ExportMod"... I will change its name.